internal/services/controller/controller.go

```diff
@@ -290,6 +296,32 @@ func (c *Controller) Run(ctx context.Context) error {
 		}, c.cfg.Interval, ctx.Done())
 	}()
 
+	metricsType := reflect.TypeOf(&v1beta1.PodMetrics{})
+	go func() {
```

Comment:
can't we wrap this logic inside "fake" informer so it would fit existing scraping nicely? At least thats the idea @saumas had when we discussed possible implementation

Reply:
Done